lib/matplotlib/cm.py

```diff
@@ -86,16 +86,13 @@ def _generate_cmap(name, lutsize):
 with _warnings.catch_warnings():
     _warnings.simplefilter("ignore")
     # Generate the reversed specifications ...
-    for cmapname in list(six.iterkeys(datad)):
-        spec = datad[cmapname]
-        spec_reversed = _reverse_cmap_spec(spec)
-        datad[cmapname + '_r'] = spec_reversed
+    for cmapname, spec in list(six.iteritems(datad)):
```

Comment:
No because we're modifying the dict at the same time; added a comment to that effect.